packages/esm-appointments-app/src/appointments/forms/create-edit-form/appointments-form.component.tsx

```diff
@@ -90,7 +90,7 @@ const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientU
   );
 
   const appointmentService = services?.find(({ uuid }) => uuid === patientAppointment.serviceUuid);
-  const today = dayjs().startOf('day').format();
+  const today = dayjs().startOf('day').format('DD/MM/YYYY');
```

Comment:
Adding a quick comment without knowing much about this ticket or the broader tickets, which I think are meant to deal with this broadly. but the *display* format of a date should be customizable by implementation.  For our default in O2, we tend to use a format "1-Jan-2024" , which the month specifically implemented as text so that "DD/MM/YYYY" is not misintrepreted as "MM/DD/YYYY" format by US users.  But you  may already be on top of that... :)  Again, I realize this is a quick fix just to get this working again.

Reply:
We can directly pass the date object in the `maxDate` prop, instead of the format, which is more reliable.
CC: @ibacher @denniskigen

Reply:
I used something similar here: https://github.com/openmrs/openmrs-esm-patient-chart/blob/f0328f86c16be660c3b06f542cd4d2a2c4fb9ae5/packages/esm-patient-chart-app/src/visit/visit-form/visit-date-time.component.tsx#L38

Reply:
Oh... yes, if the use of this variable is to populate the `maxDate` prop, we shouldn't be doing any formatting with it at all, as that will be pretty inherently non-portable.